openframe-frontend-core/src/styles/chat-animations.css

```
@@ -63,3 +63,57 @@
   .mingo-chat-content[data-state="closed"] { animation-name: mingo-chat-slide-out-right; }
 }
+
+/* =============================================================================
+ * Mingo AI header button — the 2026 AI edge-light pattern (Apple Intelligence
+ * glow-edge / Gemini gradient language): a thin accent-gradient arc TRAVELS
+ * around the button's border. No blurry blobs. Transform-only rotation on an
+ * oversized conic layer (compositor-friendly); an inset cover reveals only
+ * the edge. Reduced motion: rotation stops → a static gradient edge (the
+ * colorful state survives, only motion is removed). Hover adds a one-shot
+ * light-streak shimmer.
+ * ---------------------------------------------------------------------------*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)